.github/workflows/persistent-devnet.yml

```diff
@@ -0,0 +1,108 @@
+name: Deploy Persistent Devnet
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "Release tag, branch name or any revision."
+        required: true
+        type: string
+
+env:
+  NIX_NIXPKGS_CHANNEL: https://nixos.org/channels/nixpkgs-22.05-darwin
+  CACHIX_COMPOSABLE: composable-community
+  CACHIX_COMPRESSION_LEVEL: 3
+  NIXOPS_STATE_URL: gs://composable-state
+  NIXOPS_STATE_FILE: persistent.deployment.nixops
+
+jobs:
+  cache-devnet:
```

Comment:
name of jobs is different from display name . hard to navigate. also Persistent is already in root namespace, so having that repeated in name is not needed (same as in rust you do not do mod FooBar and then doo FooBar.FooBarMyStruct)